web/src/compose_state.js

```diff
@@ -185,3 +186,15 @@ export function has_full_recipient() {
     }
     return private_message_recipient() !== "";
 }
+
+export function update_email(user_id, new_email) {
+    let reply_to = private_message_recipient();
+
+    if (!reply_to) {
+        return;
+    }
+
+    reply_to = people.update_email_in_reply_to(reply_to, user_id, new_email);
+
+    private_message_recipient(reply_to);
+}
```

Comment:
I'll merge this to remove the import cycle, but @evykassirer do we need this function at all? I would have thought with the conversion of private message recipients to pills that everything would work with user IDs, and thus changing a user's email address should no longer require any update to `compose_state` anymore -- is that correct?

Reply:
Unless I'm forgetting something, I don't remember working on converting private message recipients to pills. My first pill work has been with search, which hasn't been merged yet. I'm happy to look into this more though.